.github/CODEOWNERS

```diff
@@ -7,15 +7,26 @@
 # For documentation on this file, see https://help.github.com/articles/about-codeowners/
 # Mentioned users will get code review requests.
 
+# Boostraping and core infra
+pkgs/stdenv/	@Ericson2314
+pkgs/build-support/cc-wrapper/	@Ericson2314
```

Comment:
Hm, who made @Ericson2314 owner of stdenv?

Reply:
Well, I believe our current use of `CODEOWNERS` is mainly for people to express interest in being pinged (automatically) about changes in some files, even though the feature [*can* be used](https://github.com/blog/2392-introducing-code-owners) in much stronger ways.

Reply:
Well, the name of the file, and [Github's description](https://help.github.com/articles/about-codeowners/) states it's clearly about ownership.

I didn't know that you could require review from code owners. That might be a good idea to enable.

Reply:
Yes, they consider it that way, but so far in nixpkgs there doesn't seem much interest in strict code ownership.  We might explicitly document our intentions at the top of the file, though it's a little risky deviating too much from the semantics envisioned by github.

Reply:
I think it would be a decent idea to use "code owners" to replace the mention bot (and I feel that had been mentioned somewhere). Rather than a strict definition of "owning" the code, it would be more in the sense of "I'd like to be invited to review PRs of this code"

Reply:
Ah, yeah, I see it as replacement of mention bot as well.
I added myself to every file I want to be notified of when it changes.

Reply:
All I wanted was notifications. I don't view it as my fief.

Reply:
@edolstra I'd be worried that if we did use it as an enforcement mechanism, then there'd be no way for non-owners to selectively subscribe to PRs. I want to be able to stay abreast of stdenv changes without needing to also convince you or anyone I should also be a final arbiter of PR approval.